pkg/apply/applier_integration_test.go

```diff
@@ -170,9 +174,14 @@ func (r *fakeReconciler) setupWithManager(mgr ctrl.Manager) error {
 	return nil
 }
 
+var mgr ctrl.Manager = nil
```

Comment:
OK - I get what you're doing here.  I've already updated this test in my checkout to wrap a bunch of contextual stuff in a `testManager` singleton, so I'd like to roll this back and merge some of your changes into my new version.  I'll submit a separate PR for that change and probably roll this file back to the previous version within this PR.

Reply:
ok